packages/aws-cdk/bin/cdk.ts

```diff
@@ -124,6 +124,12 @@ async function parseCommandLineArguments() {
         desc: 'Continuously observe the project files, ' +
           'and deploy the given stack(s) automatically when changes are detected. ' +
           'Implies --hotswap by default',
+      })
+      .options('logs', {
+        type: 'boolean',
+        default: true,
+        desc: 'Show CloudWatch log events from hotswapped resources in the terminal. ' +
+          "'true' by default, use --no-logs to turn off",
```

Comment:
We should clarify that this setting is only in effect if the `--watch` option was passed to `deploy`.